guidelines/responses.md

```diff
@@ -2,10 +2,16 @@
 
 ## Content-type
 
-The API responds with `Content-Type` set to `application/json` and JSON content. In case of success, the HTTP status code is 200 and the content contains result according to the call. In case of error, there are multiple HTTP status codes for different types of errors:
+The API responds with `Content-Type` set to `application/json` and JSON content in body. In case of 204 response, the `Content-Type` header is not set and response body is empty. 
 
 ## Response codes
 
+In case of success, the HTTP status code is either 200 and the content contains result according to the call, or the client may opt-in to receive 204 code. In case of error, there are multiple HTTP status codes for different types of errors.
```

Comment:
```suggestion
In case of success, the HTTP status code is normally 200 and the content contains the result according to the nature of the request. The client may opt-in to instead receive HTTP status code 204 in certain circumstances, see [Request minimal response](requests.md#request-minimal-response). In case of error, there are multiple HTTP status codes for different types of errors.
```